drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c

```diff
@@ -360,8 +360,13 @@ static void buffer_cb(struct vchiq_mmal_instance *instance,
 		}
```

Comment:
Should you change the commit message from "H264 header come off VC" to "H264 headers come from VC"?

Reply:
Ack